Lib/asyncio/taskgroups.py

```diff
@@ -1,4 +1,4 @@
-# Adapted with permission from the EdgeDB project.
+# Licensed to PSF from the EdgeDB project.
```

Comment:
This would seem to imply that it is unchanged from the EdgeDb version, which isn't the case at all.

Reply:
Hmm, I certainly don't want to imply that either